native/src/routes/News.tsx

```diff
@@ -8,82 +8,84 @@ import {
   fromError,
   LOCAL_NEWS_TYPE,
   LocalNewsModel,
+  NewsType,
   NotFoundError,
-  TunewsModel,
-  NEWS_ROUTE,
+  ReturnType,
   TU_NEWS_TYPE,
-  NewsType,
-  RouteInformationType
+  TunewsModel
 } from 'api-client'
 
 import Failure from '../components/Failure'
+import LoadingSpinner from '../components/LoadingSpinner'
 import NewsDetail from '../components/NewsDetail'
 import NewsList from '../components/NewsList'
 import NewsListItem from '../components/NewsListItem'
-import { NewsModelsType } from '../redux/StateType'
+
+type NewsModelsType = Array<LocalNewsModel | TunewsModel>
 
 const NoNews = styled.Text`
   color: ${props => props.theme.colors.textColor};
   font-family: ${props => props.theme.fonts.native.contentFontRegular};
   align-self: center;
   margin-top: 20px;
 `
-export type PropsType = {
+export type PropsType = ReturnType<NewsModelsType> & {
+  selectNews: (newsId: string | null) => void
   newsId: string | null | undefined
-  news: NewsModelsType
   cityModel: CityModel
   language: string
   selectedNewsType: NewsType
-  isFetchingMore: boolean
-  fetchMoreNews: () => void
-  navigateTo: (arg0: RouteInformationType) => void
-  routeKey: string
+  loadMore?: () => void
+  loadingMore?: boolean
```

Comment:
Don't see where this is ever set?

Reply:
Set by passing the whole response of `useLoadTuNews` to `News`